ormconfig.js

```diff
@@ -1,20 +1,21 @@
 const pgconnection = require("pg-connection-string")
 require('dotenv').config()
 
+let connectionOptions = ''
 const databaseUrl = process.env.DATABASE_URL
-const connectionOptions = pgconnection.parse(databaseUrl)
 
-module.exports = {
+if (databaseUrl){
+  connectionOptions = pgconnection.parse(databaseUrl)
+}
+
+let connectionConfig = {
    "name": 'default',
    "type": "postgres",
-   "host": connectionOptions.host,
-   "port": connectionOptions.port,
-   "extra": {
-      ssl: { rejectUnauthorized: false }
- },
-   "username": connectionOptions.user,
-   "password": connectionOptions.password,
-   "database": connectionOptions.database,
+   "host": connectionOptions.host || 'localhost',
+   "port": connectionOptions.port || 5432,
+   "username": connectionOptions.user || 'postgres',
```

Comment:
Should we change 'postgres' to process.env.DB_USER? This may error or cause merge issues on some local machines.

Reply:
Good catch!